binder/form.go

```diff
@@ -51,8 +53,8 @@ func (b *formBinding) Bind(reqCtx *fasthttp.RequestCtx, out any) error {
 }
 
 // BindMultipart parses the request body and returns the result.
-func (b *formBinding) BindMultipart(reqCtx *fasthttp.RequestCtx, out any) error {
-	data, err := reqCtx.MultipartForm()
+func (b *FormBinding) BindMultipart(req *fasthttp.Request, out any) error {
```

Comment:
maybe 2 different binders, then you could make a unified binder interface and the formMultipart binder could have a `Bind` method